### DISCUSSION TITLE
Unable to deploy Tanzu Management Cluster

### DISCUSSION THREAD

**Discussion by prbjrgac** [vmware-tanzu/community-edition#4953] [General]
Been trying to get the Tanzu Management Cluster installed and keep getting the error listed below. Have followed the pre-requisites and believe it should be good to.. Any suggestions would be greatly appreciated. 

Running Ubuntu 20.04

![Screenshot 2022-07-01 131620](https://user-images.githubusercontent.com/37939115/176956683-51de0ece-fc4b-4866-b424-3a8c8f2c2012.png)

**Comment by stmcginnis** [comment]
Please see the troubleshooting steps here: https://tanzucommunityedition.io/docs/v0.12/ts-manually/

If you look at the pod status and logs, it should likely point to the reason for the failure.
